starters/basic-starter/components/node--article.tsx

```diff
@@ -3,13 +3,81 @@ import Link from 'next/link';
 import { formatDate } from 'lib/format-date';
 import { MediaImage } from 'components/media--image';
 import { FormattedText } from 'components/formatted-text';
+import React, { useState } from 'react';
+import { drupal } from '../lib/drupal';
 
-export function NodeArticle({ node, ...props }) {
+const initialValues = {
+  name: '',
+  email: '',
+  subject: '',
+  message: '',
+};
+
+export function NodeArticle({ node, additionalContent, ...props }) {
+  const [values, setValues] = useState(initialValues);
+
+  function handleInputChange(
+    e:
+      | React.ChangeEvent<HTMLInputElement>
+      | React.ChangeEvent<HTMLTextAreaElement>,
+  ) {
+    const { name, value } = e.target;
+    setValues({
+      ...values,
+      [name]: value,
+    });
+    console.log(e.target);
+  }
+
+  // Commenting this function out will make page render
+  async function submitForm() {
+    const baseUrl = await drupal.baseUrl;
```

Comment:
Running into this error with the drupal client. Tried to create a new client and also tried to move it out of this file to a file that has an existing `drupal` but still getting this error. Not sure why since it's being given the clientId and clientSecret.

![image](https://user-images.githubusercontent.com/35243020/181125648-9051df40-50a1-49e7-9507-f3f99e4e1d65.png)